src/librustc/traits/specialize/specialization_graph.rs

```diff
@@ -467,32 +496,18 @@ impl<T> NodeItem<T> {
 }
 
 impl<'tcx> Ancestors<'tcx> {
-    /// Search the items from the given ancestors, returning each definition
-    /// with the given name and the given kind.
-    // FIXME(#35870): avoid closures being unexported due to `impl Trait`.
```

Comment:
Is this fixme resolved / rendered moot?

Reply:
Yes, https://github.com/rust-lang/rust/issues/35870 was fixed by #43857